service/src/main/java/bio/terra/user/service/iam/SamRethrow.java

```diff
@@ -0,0 +1,60 @@
+package bio.terra.user.service.iam;
+
+import bio.terra.common.sam.exception.SamExceptionFactory;
```

Comment:
you don't *have* to do this but there are at least three copies of SamRethrow in Databiosphere now.. https://cs.github.com/?scopeName=All+repos&scope=&q=samrethrow

it should live with SamRetry here https://cs.github.com/DataBiosphere/terra-common-lib/blob/84d0db138bd713a339391519686308e160b58187/src/main/java/bio/terra/common/sam/SamRetry.java?q=samretry

Reply:
Just to confirm, you're suggesting contributing SamRethrow to the common lib right? If so I agree and was surprised to find it not there already